src/components/interfaces/MOBILE_API.xml

```diff
@@ -4112,9 +4116,7 @@
         <param name="myKey" type="Boolean" mandatory="false">
             <description>Information related to the MyKey feature</description>
         </param>
-        
         <!-- / Ford Specific Data Items -->
-        
```

Comment:
Unrelated style changes, unfortunately these will not be kept unless these same changes are submitted to the https://github.com/smartdevicelink/rpc_spec. The final spec will be directly copied to this Repo prior to the next release. As such, they should not be present here.